File: NetCoreApi2/clientApp/src/features/proyectos/models/proyectos.ts
export interface ProyectosInterface{
    id: number;
    nombre: string;
    clientesId: number;
}//end interface

export class ProyectosModel{
    id: number;
    nombre: string;
    clienteId: number;

    constructor(data: ProyectosInterface){
        this.id = data.id;
        this.nombre = data.nombre;
        this.clienteId = data.clientesId;
    }//end constructor
}//end class ProyectosModel

File: NetCoreApi2/Data/Models/Proyectos.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace NetCoreApi2.Data.Models
{
    public class ProyectosModel
    {
        [Key]
        public int Id { get; set; }
        public string Nombre { get; set; }
        public int ClienteId { get; set; }  
        public ClientesModel cliente { get; set; } 
        public ICollection<PartesModel> Partes { get; set; }

    }
}
